### DISCUSSION TITLE
enhancement: NPCs could know that they can't reach someone

### DISCUSSION THREAD

**Discussion by Daywalkrr** [AscensionGameDev/Intersect-Engine#1553] [Ideas]
**Is your feature request related to a problem? Please describe.**
It would be cool if NPCs could know when they _couldn't_ reach someone, so they don't trigger aggro. This could also be a configurable option on each NPC for NPCs that you're okay with aggroing through walls or whatever.

**Describe the solution you'd like**
A checkbox on NPCs, "ignore walls for aggro" or something, and otherwise have NPCs that can't reach a player prevent aggroing on that player unless otherwise hit by the player.

**Comment by da-rulez** [comment]
Very nice suggestion. 

Ideal for NPC's that are below you when standing on a cliff or bridge.

**Comment by SilverShardOnline** [comment]
This would be wonderful.
